datatypes/src/jvmMain/kotlin/at/asitplus/crypto/datatypes/JcaExtensions.kt

```diff
@@ -14,12 +18,16 @@ import org.bouncycastle.jce.spec.ECPublicKeySpec
 import java.math.BigInteger
 import java.security.KeyFactory
 import java.security.PublicKey
+import java.security.cert.CertificateFactory
 import java.security.interfaces.ECPublicKey
 import java.security.interfaces.RSAPublicKey
 import java.security.spec.MGF1ParameterSpec
 import java.security.spec.PSSParameterSpec
 import java.security.spec.RSAPublicKeySpec
 
+private val crtFactMut = Mutex()
```

Comment:
longer variable names do not hurt nobody :-)

Reply:
i'll change it